resources/views/slack-invite.blade.php

```diff
@@ -39,7 +39,7 @@
 
         <div class="container text-center">
 
-            <h2>Join {{ config('sitedata.slack_team_name') }} on Slack</h2>
+            <h2>Join &ldquo;{{ ucwords(strtolower(config('sitedata.slack_team_name'))) }}&rdquo; on Slack</h2>
```

Comment:
Team name display is now formatted such that words are capitalised and the team name is placed in quotes.